sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsActions.cs

```diff
@@ -14,27 +14,27 @@ public class TextAnalyticsActions
         public string DisplayName { get; set; }
 
         /// <summary>
-        /// Extract KeyPhrases actions.
+        /// The set of "Extract KeyPhrases Actions" that will get executed on the input documents.
```

Comment:
in case of a rename or that something changes, you could do `The set of <see cref="ExtractKeyPhrasesAction"/> that will...` and that way a link gets created and any rename will be done automatically